client/sites/tools/staging-site/components/staging-site-card/staging-site-production-card.tsx

```diff
@@ -113,6 +129,7 @@ function StagingSiteProductionCard( { disabled, siteId, translate }: CardProps )
 					<Button
 						primary
 						onClick={ () => {
+							dispatch( recordTracksEvent( 'calypso_hosting_configuration_staging_site_to_prod' ) );
```

Comment:
Would it make sense to track `calypso_hosting_staging_site_manage` and `calypso_hosting_configuration_staging_site_to_prod` in a similar way to `calypso_sites_environment_switched`, where the event is the same but parameters are different?